package.json

```diff
@@ -30,12 +30,13 @@
     "aries-site"
   ],
   "dependencies": {
-    "grommet": "https://github.com/grommet/grommet/tarball/stable",
+    "grommet": "https://github.com/grommet/grommet/tarball/styled-components-beta-stable",
     "grommet-icons": "https://github.com/grommet/grommet-icons/tarball/stable",
     "grommet-theme-hpe": "https://github.com/grommet/grommet-theme-hpe/tarball/stable",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "styled-components": "^5.3.6"
+    "styled-components": "^6.1.0",
+    "stylis": "^4.3.0"
```

Comment:
Do we need to add stylis? I thought styled-components includes this directly.

Reply:
Yup, forgot we don't need to include this. Good call out